internal/core/metadata/operators/device_profile/value_descriptor.go

```diff
@@ -3,11 +3,13 @@ package device_profile
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
 
 	"github.com/edgexfoundry/edgex-go/internal/core/data/errors"
+	errors2 "github.com/edgexfoundry/edgex-go/internal/core/metadata/errors"
```

Comment:
Maybe what we need to consider is moving all error type definitions to internal/pkg/errors to avoid this kind of aliasing. Separate issue, of course.